Makefile

```
@@ -304,32 +304,52 @@ stack-restart: dev-restart ## alias for dev-restart
 
-ci-runner-stop: ## stop the self-hosted CI runner
-	@test -f "$(CI_RUNNER_DIR)/svc.sh" || exit 0
-	@cd "$(CI_RUNNER_DIR)" && sudo ./svc.sh stop
+ci-runner-stop: ## stop the self-hosted CI runners
+	@for dir in $(CI_RUNNER_DIRS); do \
+		test -f "$$dir/.service" || continue; \
+		(cd "$$dir" && sudo ./svc.sh stop) || exit 1; \
+	done
 
-ci-runner-restart: ci-runner-stop ci-runner-start ## restart the self-hosted CI runner
+ci-runner-restart: ci-runner-stop ci-runner-start ## restart the self-hosted CI runners
```

checkmake (0.3.2)

[warning] Target "ci-runner-restart" should be declared PHONY.

(phonydeclared)